pom.xml

```diff
@@ -118,43 +118,43 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>co.cask.cdap</groupId>
+      <groupId>io.cdap.cdap</groupId>
       <artifactId>cdap-proto</artifactId>
       <version>${cdap.version}</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>co.cask.cdap</groupId>
+      <groupId>io.cdap.cdap</groupId>
       <artifactId>cdap-formats</artifactId>
       <version>${cdap.version}</version>
       <scope>provided</scope>
```

Comment:
existing problem, but this shouldn't be provided. If classes from cdap-formats are required, they need to be packaged with the plugin.

This also makes me wonder whether this dependency is actually needed.

Reply:
removed this dependency